client-src/elements/chromedash-feature.ts

```diff
@@ -144,8 +153,8 @@ class ChromedashFeature extends LitElement {
       target.tagName == 'A' ||
       target.tagName == 'CHROMEDASH-MULTI-LINKS' ||
       e.composedPath()[0].nodeName === 'A' ||
-      textSelection.type === 'RANGE' ||
-      textSelection.toString()
+      (textSelection &&
+        (textSelection.type === 'RANGE' || textSelection.toString()))
```

Comment:
This looks like it'll do the same thing:
```suggestion
      textSelection?.type === 'RANGE' ||
      textSelection?.toString())
```